starter-code/lib/lodash.js

```diff
@@ -0,0 +1,21 @@
+var opinions = [ "This is the best job ever",
+                 "Satisfied",
+                 "At least I get paid",
+                 "I'm looking for another job",
+                 "I don't want to answer"];
+
+var database = [];
+
+function pickOpinion() {
+  return _.sample(opinions);
+}
+
+function intoDatabase(){
+  database.push(_.times(10, pickOpinion));

```

Comment:
no need to call push because `times` returns an array already

```js
function intoDatabase () {
  return _.times(10, pickOpinion);
}

var database = intoDatabase()
```